### DISCUSSION TITLE
how to disable user registration and manage by admin only

### DISCUSSION THREAD

**Discussion by prafullbyjus** [Flagsmith/flagsmith#518] [Q&A] [answered]
I want to disable registration from frontend. administrator can login using provided username and password and only admin can add/manage the rest of user management. I tried to setup using api/app/settings/common.py file but not reflecting our changes on pages. Thanks in advance

**Comment by sdenovan** [comment, marked as answer]
Set the environment variable `PREVENT_SIGNUP=true`
